lib/importer.js

```diff
@@ -65,13 +99,13 @@ var emptyChangesSummary = function(lastSeq) {
   };
 };
 
-var importChangesBatch = function(db, couchdb, concurrentDocLimit, changesLimit) {
-  return db.one('SELECT seq FROM couchdb_progress')
-    .then(function(seqResult) {
-      log.debug('Downloading CouchDB changes feed from ' + seqResult.seq);
+var importChangesBatch = function(db, couchdb, concurrentDocLimit, changesLimit, source) {
+  return getSeq(db, source)
```

Comment:
If you refer to the version exported in the module you can mock it out, which may or may not make your tests easier :-)